apps/modernization-ui/src/apps/search/layout/SearchLayout.tsx

```diff
@@ -10,7 +10,7 @@ import { SearchNavigation } from './navigation/SearchNavigation';
 
 type Renderer = () => ReactNode;
 
-type Props = {
+export type Props = {
```

Comment:
question (non-blocking): What needed this type to be exported?

Reply:
I was using it somewhere before but with the recent changes I removed it so I am reverting this back